package.json

```diff
@@ -56,7 +56,8 @@
         "onCommand:extension.draftCreate",
         "onCommand:extension.draftUp",
         "onView:extension.vsKubernetesExplorer",
-        "onLanguage:helm"
+        "onLanguage:helm",
+        "onLanguage:yaml"
```

Comment:
Do we need to activate on arbitrary YAML files?  Are we providing Helm intellisense on arbitrary YAML as well as on files that match the Helm language filter?

Reply:
We need onLanguage:yaml because kubernetes  manifest files are yaml language, we will be activated to be able to call registerYamlSchemaSupport, otherwise, user will be very confuse because activating the extension or not will have different behaviors for kubernetes manifest files when sometimes he see the hoved document and sometimes not(because our extension is not activated )

Reply:
for your question: Are we providing Helm intellisense on arbitrary YAML as well as on files that match the Helm language filter
on arbitrary YAML , if user has chosen yaml as the language, then we are providing Helm intellisense only when yaml is under `templates` folder(we can add more filters after detecting the helm/chart folder structure later). On the other side, we try to provide kubernetes manifest support (validation/auto-completion) only when we recognize it as kubernetes manifest support by kind and apiVersion